frontend/src/User/Login.tsx

```diff
@@ -38,6 +39,9 @@ export default function Login() {
 
     return (
         <div>
+            <div className={'navBar'}>
+                <NavBar/>
+            </div>
```

Comment:
```suggestion
                <NavBar/>
```
Since the navbar already has an div with this classname it's redundant